src/sentry/runner/commands/run.py

```diff
@@ -745,3 +745,12 @@ def last_seen_updater(**options):
 
     with global_tags(_all_threads=True, pipeline=ingest_config.internal_metrics_tag):
         run_processor_with_signals(consumer)
+
+
+@run.command()
```

Comment:
Let's give this an explicit name to follow our convention of dashes.

```suggestion
@run.command("backpressure-monitor")
```